ninja-core/src/main/java/ninja/migrations/flyway/MigrationEngineFlyway.java

```diff
@@ -41,12 +41,19 @@ public void migrate() {
         String connectionUrl = ninjaProperties.getOrDie(NinjaConstant.DB_CONNECTION_URL);
         String connectionUsername = ninjaProperties.getOrDie(NinjaConstant.DB_CONNECTION_USERNAME);
         String connectionPassword = ninjaProperties.getOrDie(NinjaConstant.DB_CONNECTION_PASSWORD);
+        String locations = ninjaProperties.get(NinjaConstant.NINJA_MIGRATION_LOCATIONS);
+        String schemas = this.ninjaProperties.get(NinjaConstant.NINJA_MIGRATION_SCHEMAS);
 
         // We migrate automatically => if you do not want that (eg in production)
         // set ninja.migration.run=false in application.conf
         Flyway flyway = new Flyway();
         flyway.setDataSource(connectionUrl, connectionUsername, connectionPassword);
 
+        if (locations != null)
```

Comment:
very tiny, but could you add curly braces to both if statements?

Reply:
Done.